include/mbedtls/aes.h

```diff
@@ -298,34 +298,35 @@ int mbedtls_aes_crypt_cfb8( mbedtls_aes_context *ctx,
- *        or decryption operation.
- *
- *        For OFB, you must set up the context with mbedtls_aes_setkey_enc(),
- *        regardless of whether you are performing an encryption or decryption
- *        operation. This is because OFB mode uses the same key schedule for
- *        encryption and decryption.
- *
- *        The OFB operation is identical for encryption or decryption, therefore
- *        no operation mode needs to be specified.
- *
- * \note  Upon exit, the content of iv, the Initialisation Vector, is updated
- *        so that you can call the same function again on the next block(s) of
- *        data and get the same result as if it was encrypted in one call. This
- *        allows a "streaming" usage, by initialising iv_off to 0 before the
- *        first call, and preserving its value between calls.
- *
- *        For block by block usage, (or non-streaming use), the iv should be
- *        initialised on each call to a unique value, and iv_off set to 0 on
- *        each call.
- *
- *        If you need to retain the contents of the initialisation vector, you
- *        must either save it manually or use the cipher module instead.
- *
- *        For the OFB mode, the initiallisation vector must be unique and must
- *        be unique for every encryption operation. Reuse of an initialisation
- *        vector will compromise security.
- *
+ * \brief       This function performs an AES-OFB (Output Feedback Mode)
+ *              encryption or decryption operation.
+ *
+ *              For OFB, you must set up the context with
+ *              mbedtls_aes_setkey_enc(), regardless of whether you are
+ *              performing an encryption or decryption operation. This is
+ *              because OFB mode uses the same key schedule for encryption and
+ *              decryption.
+ *
+ *              The OFB operation is identical for encryption or decryption,
+ *              therefore no operation mode needs to be specified.
+ *
+ * \note        Upon exit, the content of iv, the Initialisation Vector, is
+ *              updated so that you can call the same function again on the next
+ *              block(s) of data and get the same result as if it was encrypted
+ *              in one call. This allows a "streaming" usage, by initialising
+ *              iv_off to 0 before the first call, and preserving its value
+ *              between calls.
+ *
+ *              For non-streaming use, the iv should be initialised on each call
+ *              to a unique value, and iv_off set to 0 on each call.
+ *
+ *              If you need to retain the contents of the initialisation vector,
+ *              you must either save it manually or use the cipher module
+ *              instead.
+ *
+ * \warning     For the OFB mode, the initiallisation vector must be unique and
+ *              must be unique for every encryption operation. Reuse of an
```

Comment:
I tried to ask earlier but I assume it was lost in the middle of other points: Is the repetition of "must be unique" a typo or is their some subtlety I'm missing in the sentence structure?

Reply:
Maybe it's poor phrasing. The wording is trying to say the IV must be unique, and and it needs to be unique for each successive encryption operation.

And now you highlight it, I can see 'initiallisation' is misspelt.

Reply:
I've pushed a commit to make the wording more clear. I also fixed the misspelling.

Reply:
Thanks! I would have done it myself if I'd had time.